geozero/src/geos/geos_reader.rs

```diff
@@ -25,7 +25,9 @@ impl From<geos::Error> for GeozeroError {
             | geos::Error::GeosError(e)
             | geos::Error::NoConstructionFromNullPtr(e)
             | geos::Error::ConversionError(e)
-            | geos::Error::GenericError(e) => GeozeroError::Geometry(e),
+            | geos::Error::GenericError(e)
+            | geos::Error::NormalizeError(e)
```

Comment:
sorry, a bit unclear - where did this change came from?

Reply:
a `clippy` error, un-handled variants. Should I remove?

Reply:
ahem, it shouldn't even compile if some variants are not listed -- match is exhaustive... i'm confused now

Reply:
for clarity, lets remove this and keep it in mind for a subsequent PR

Reply:
FWIW, I'm was also getting this error when running `clippy` locally:

```sh
error: you are implementing `Hash` explicitly but have derived `PartialEq`
  --> geozero/src/mvt/tile_value.rs:80:1
   |
80 | / impl Hash for TileValue {
81 | |     fn hash<H: std::hash::Hasher>(&self, state: &mut H) {
82 | |         match self {
83 | |             Self::Str(s) => s.hash(state),
...  |
91 | |     }
92 | | }
   | |_^
   |
note: `PartialEq` implemented here
  --> geozero/src/mvt/tile_value.rs:5:24
   |
5  | #[derive(Debug, Clone, PartialEq)]
   |                        ^^^^^^^^^
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#derive_hash_xor_eq
   = note: `#[deny(clippy::derive_hash_xor_eq)]` on by default
   = note: this error originates in the derive macro `PartialEq` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `geozero` due to previous error
```

but I wasn't sure how to fix it...

I've got `clippy 0.1.67 (d5a82bb 2023-02-07)`

Reply:
:thinking: i'll take a look at it, thx

Reply:
Sigh, this is a breaking upstream change that was done as a patch change. https://github.com/georust/geos/pull/133#issuecomment-1603243591

I will fix it in a separate PR